doc/rm/comportability_specification/index.md

```diff
@@ -359,6 +415,35 @@ The following shows the expected documentation format for this example.
 | `fatal_uart_breach` | Someone has attacked the UART module |
 | `recov_uart_frozen` | The UART lines are frozen and might be under attack |
 
+### Specifying countermeasures
+
+Countermeasure information can be specified in the register hjson format.
+This is done with a list with key `countermeasures`.
+Each item is a dictionary with keys `name` and `desc`.
+The `desc` field is a human-readable description of the countermeasure.
+The `name` field should be either of the form `ASSET.CM_TYPE` or `INSTANCE.ASSET.CM_TYPE`.
+Here, `ASSET` and `CM_TYPE` should be one of the values given in the tables in the [Security countermeasures]({{< relref "#countermeasures" >}}) section.
+If specified, `INSTANCE` should name a part of the IP block holding the asset.
```

Comment:
I'd rather say `submodule`, otherwise this could be misinterpreted (we sometimes refer to comportable IPs as IP blocks).

Reply:
Do you mean "... a submodule of the IP block ..." ? (I'm not quite sure what renaming you mean!)

Reply:
Sorry for being unspecific. 
Also I realized that I misread this sentence: You already mention that it `should name a part of the IP block`, which I think covers what I had in mind.

So nevermind and sorry for the spam :).